tests/util.go

```diff
@@ -45,6 +45,11 @@ const (
 	paramFlag          = "-p"
 )
 
+const (
+	ShortTimeout       = time.Duration(2) * time.Minute
```

Comment:
This variable is not used, all the tests have a timeout of 4 min now.
Maybe it will be better to change the execute function to receive a timeout variable and configure the timeout on every test to get a better control over the test run time.

Reply:
I agree with you, but I would like to go with this approach in a following PR.
Currently, every second PR is failing because of a timeout, and we can't  this situation.

Reply:
Sounds good!